.github/containers/ruby/repo/packages/py-hdbscan/package.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack.package import *
+
+
+class PyHdbscan(PythonPackage):
```

Comment:
Why do we need this package? Who is using it?

Reply:
needed by dependency